vita3k/shader/src/spirv_recompiler.cpp

```diff
@@ -748,7 +748,7 @@ static SpirvShaderParameters create_parameters(spv::Builder &b, const SceGxmProg
         }
     }
 
-    spv::Id memory_type = b.makeArrayType(f32_type, b.makeIntConstant(total_buffer_size * 4), 4);
+    spv::Id memory_type = b.makeArrayType(f32_type, b.makeIntConstant(total_buffer_size * 4 + 100), 0);
```

Comment:
why + 100 ?

Reply:
oops.

Reply:
Should be +1. Padding to prevent oob error in https://github.com/Vita3K/Vita3K/pull/997/files#diff-67cafa4ae93b4e97d96369e165fd84273388603a50fb8c4d36fb01df1dcdc75dR403.

Reply:
Fixed